wp-admin/js/widgets/media-image-widget.js

```diff
@@ -25,17 +25,41 @@
 	ImageWidgetControl = component.MediaWidgetControl.extend({
 
 		/**
+		 * Get the appropriate image url for the preview.
+		 *
+		 * @returns {string} Image url
+		 */
+		getImagePreviewUrl: function getImagePreviewUrl() {
+			var control = this, attachmentId, size, attachmentSizes;
+			attachmentId = control.model.get( 'attachment_id' );
+			size = control.model.get( 'size' );
+
+			// Attempt to return the specific attachment image size requested.
+			attachmentSizes = control.selectedAttachment.get( 'sizes' );
+			if ( attachmentId && 'custom' !== size && attachmentSizes && attachmentSizes[ size ] ) {
+				return attachmentSizes[ size ].url;
+			}
```

Comment:
@timmyc I think in practice this will always end up being the same as the `control.model.get( 'url' )` since the URL for the specific size chosen: https://github.com/xwp/wp-core-media-widgets/blob/0502cd6d7a04fa5bd74fb21bae56a3a3cb38dcaf/wp-admin/js/widgets/media-widgets.js#L516-L518

So I wonder if we need this function and an just replace calls to `control.getImagePreviewUrl()` with simply `control.model.get( 'url' )`?